vowpalwabbit/io/logger.h

```diff
@@ -60,62 +60,74 @@ namespace logger
     off = spdlog::level::off
   };
 
+  // FIXME: the get() call returns a shared_ptr. Keep a copy here to avoid unnecessary shared_ptr copies
+  // This can go away once we move to an object-based logger
+  namespace detail
+  {
+  extern std::shared_ptr<spdlog::logger> _stderr_logger;
+  extern size_t max_limit;
+  extern size_t log_count;
```

Comment:
I believe these should be static rather than extern

Reply:
synced offline: can't be static since we need only one instance, this file gets included in multiple translation units